drivers/bluetooth/hci/userchan.c

```diff
@@ -56,6 +58,12 @@ static int uc_fd = -1;
 
 static int bt_dev_index = -1;
 
+#define TCP_ADDR_BUFF_SIZE 16
+static bool hci_socket;
+static char ip_addr[TCP_ADDR_BUFF_SIZE] = {0};
```

Comment:
```suggestion
static char ip_addr[TCP_ADDR_BUFF_SIZE];
```

Since it is `static`, the initialization here does nothing